website/static/js/share/utils.js

```diff
@@ -31,6 +31,12 @@ var loadMore = function(vm) {
         return;
     }
     var page = vm.page++ * 10;
+    var sort;
+    if (vm.sort() === 'Date') {
```

Comment:
We might want to make the externally facing "Date" more descriptive, like "date updated at source" or something, but less verbose - because these usually aren't publication dates or anything obvious. Thoughts?